scripts/pre_commit_hook.py

```diff
@@ -139,13 +110,21 @@ def main():
         sys.exit(0)
 
     python_utils.PRINT('Running pre-commit check for package-lock.json ...')
-    if _does_diff_include_package_lock_file_and_no_package_file():
+    if _does_diff_include_package_lock_file():
+        # The following message is necessary since there git commit aborts
+        # quietly when the status is non-zero.
+        python_utils.PRINT('-----------COMMIT ABORTED-----------')
+        python_utils.PRINT(
+            'Oppia utilize Yarn to manage node packages. Please use Yarn '
+            'to add, update, or delete packages. For more information on how '
+            'to use yarn, see https://yarnpkg.com/en/docs/usage.'
+        )
         python_utils.PRINT(
-            'This commit only changes package-lock.json without '
-            'any change in package.json. Therefore changes in '
-            'package-lock.json will be automatically reverted.')
-        python_utils.PRINT('Reverting changes in package-lock.json ...')
-        _revert_changes_in_package_lock_file()
+            'Please delete package-lock.json, revert the changes in '
+            'package.json, and use yarn to add, update, or delete the '
+            'packages.'
+        )
```

Comment:
This comment kinda repeats the previous one, might be worth to rephrase.

Reply:
done!